build_tools/stm32-cmsis-base.ld

```diff
@@ -239,6 +245,15 @@ SECTIONS
   } > RAM2
 */
 
+  .got :
+  {
+    __got_start__ = .;
+    *(.got)
+    *(.got.*)
```

Comment:
I've a patch ready that copies the `.got.plt` separately. Let's discuss this offline.